db/mysql.go

```diff
@@ -0,0 +1,57 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var DB *sql.DB
+
+func InitDB() {
+	var err error
+	DB, err = sql.Open("mysql", "root:Jason@123@tcp(127.0.0.1:3306)/restaurant")
+	if err != nil {
+		panic(err.Error())
+	}
+
+	err = DB.Ping()
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("==========Successfully connected to the database==========")
+}
+
+func CloseDB() {
+	DB.Close()
+}
+
+func InsertDB() {
+	insert, err := DB.Query("INSERT INTO restaurant.vendor VALUES ( 5, 'vendor_name', 'test', '10:00-15:00')")
+	if err != nil {
+		panic(err.Error())
+	}
+	defer insert.Close()
+}
+
+func GetDB() {
```

Comment:
function name for Get should be more descriptive: func RetrieveVendors(cond Conditions)
for get, can consider making a struct for where conditions